code/modules/clothing/masks/hailer.dm

```diff
@@ -1,10 +1,56 @@
+// **** Dispatch ****
+
+GLOBAL_LIST_EMPTY(sechailers)
+
+/datum/action/item_action/dispatch
+	name = "Signal dispatch"
+	desc = "Opens up a quick select wheel for reporting crimes, including your current location, to your fellow security officers."
+	button_icon_state = "dispatch"
+	icon_icon = 'icons/mob/actions/hailer_actions.dmi'
+
+/obj/item/clothing/mask/gas/sechailer
+	var/obj/item/radio/radio
+	var/radio_key = /obj/item/encryptionkey/headset_sec
+	var/radio_channel = "Security"
+	var/dispatch_cooldown = 20 SECONDS
+	var/last_dispatch = 0
+
+/obj/item/clothing/mask/gas/sechailer/Destroy()
+	QDEL_NULL(radio)
+	QDEL_NULL(radio_key)
```

Comment:
You can't delete a typepath

Reply:
No clue how I would resolve this part then.